lib/stat_tracker.rb

```diff
@@ -30,4 +30,16 @@ def all_teams
       team
     end
   end
+
+  def highest_total_score
+    all_games.map do |game|
+      game.score_count
+    end.max
+  end
+
```

Comment:
Do you think the enumerables .max_by and .min_by would do the same thing as .map + .max/min ?

Reply:
@Logan and I together saw that max_by and min_by returns the object and not the score of the max/min